docs/content.zh/docs/ops/state/checkpoints_vs_savepoints.md

```diff
@@ -24,72 +24,65 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Checkpoints vs. Savepoints
+<a name="checkpoints-vs-savepoints"></a>
 
-## Overview
+# Checkpoints 与 Savepoints
 
-Conceptually, Flink's [savepoints]({{< ref "docs/ops/state/savepoints" >}}) are different from [checkpoints]({{< ref "docs/ops/state/checkpoints" >}}) 
-in a way that's analogous to how backups are different from recovery logs in traditional database systems.
+<a name="overview"></a>
 
-The primary purpose of checkpoints is to provide a recovery mechanism in case of unexpected job failures. 
-A [checkpoint's lifecycle]({{< ref "docs/dev/datastream/fault-tolerance/checkpointing" >}}) is managed by Flink, 
-i.e. a checkpoint is created, owned, and released by Flink - without user interaction. 
-Because checkpoints are being triggered often, and are relied upon for failure recovery, the two main design goals for the checkpoint implementation are 
-i) being as lightweight to create and ii) being as fast to restore from as possible. 
-Optimizations towards those goals can exploit certain properties, e.g., that the job code doesn't change between the execution attempts. 
+## 概述
+
+从概念上讲，Flink 的 [savepoints]({{< ref "docs/ops/state/savepoints" >}}) 与 [checkpoints]({{< ref "docs/ops/state/checkpoints" >}}) 的不同之处类似于传统数据库系统中的备份与恢复日志之间的差异。
+
+Checkpoints 的主要目的是为意外失败的作业提供恢复机制。
+[Checkpoint 的生命周期]({{< ref "docs/dev/datastream/fault-tolerance/checkpointing" >}}) 由 Flink 管理，
+即 Flink 创建，管理和删除 checkpoint - 无需用户交互。
+作为一种恢复和定期触发的方法，Checkpoint 实现有两个设计目标：i）轻量级创建和 ii）尽可能快地恢复。 可能会利用某些特定的属性来达到这个，例如， 工作代码在执行尝试之间不会改变。
```

Comment:
```suggestion
由于 checkpoint 被经常触发，且被用于作业恢复，所以 Checkpoint 的实现有两个设计目标：i）轻量级创建和 ii）尽可能快地恢复。 可能会利用某些特定的属性来达到这个目标，例如， 作业的代码在执行尝试时不会改变。
```

Reply:
Thanks @1996fanrui , your suggestion really better, committed it.